steps/xcode-test-bundle-builder/0.0.3/step.yml

```diff
@@ -0,0 +1,81 @@
+title: Build xcode test runner for remote testing service
+summary: Build your XCTest or XCUITest runner to upload to remote testing service
+description: Build your XCTest or XCUITest bundle to upload to remote testing service
```

Comment:
managed to mark the wrong line.. - should be https://github.com/bitrise-io/bitrise-steplib/pull/694/files#diff-5517e5d4663d90868de4ed46bff4a32fR4